### PR TITLE
Klarna Setup - Update for 2.4 release

### PR DESCRIPTION
**Pull request by samoko85**
## Purpose of this pull request

Updated for 2.4 release.

## Affected documentation pages

https://docs.magento.com/user-guide/payment/klarna-setup.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/payment/klarna-setup.md [hunk 1]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
 1. When complete, click <span class="btn">Save Config</span>.
 
 1. Click [**Cache Management**]({% link system/cache-management.md %}) in the message at the top of the page, and refresh all invalid caches.
+
+## Step 4: Configure On-Site Messaging (Optional)
+
+1. Activate On-Site Messaging from you Klarna Merchant Portal by following this [guide](https://developers.klarna.com/documentation/on-site-messaging/integration/platform-activation/).
```
Comment (ccondra) [inline src/payment/klarna-setup.md:125]:
```suggestion
1. Activate On-Site Messaging from your Klarna Merchant Portal by following this [guide](https://developers.klarna.com/documentation/on-site-messaging/integration/platform-activation/).
```

File: src/payment/klarna-setup.md [hunk 2]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
...
 1. Click [**Cache Management**]({% link system/cache-management.md %}) in the message at the top of the page, and refresh all invalid caches.
+
+## Step 4: Configure On-Site Messaging (Optional)
+
+1. Activate On-Site Messaging from your Klarna Merchant Portal by following this [guide](https://developers.klarna.com/documentation/on-site-messaging/integration/platform-activation/).
+2. Retrieve your **Data Client ID**
+3. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Klarna On-Site Messaging** section.
+4. Set **Enable** to `Yes`.
+5. Enter the **data-client-id** obtained in step 1.
+6. Choose a **Design theme** between **default** or **dark**
```
Comment (ccondra) [inline src/payment/klarna-setup.md:130]:
```suggestion
6. Choose a **Design theme** between **default** or **dark**.
```

Comment (ccondra) [reply src/payment/klarna-setup.md:130]: Fixed punctuation.

File: src/payment/klarna-setup.md [hunk 3]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
...
+7. To enable OSM in the product page set **Enable on Product** to **Yes** and select a **Product placement**
```
Comment (ccondra) [inline src/payment/klarna-setup.md:131]:
```suggestion
7. To enable OSM in the product page set **Enable on Product** to `Yes` and select a **Product placement**.
```

Comment (ccondra) [inline src/payment/klarna-setup.md:131]:
```suggestion
7. To enable OSM in the product page, set **Enable on Product** to **Yes** and select a **Product placement**
```

File: src/payment/klarna-setup.md [hunk 4]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
...
+8. To enable OSM in the cart page set **Enable on Cart** to **Yes** and select a **Cart placement**
```
Comment (ccondra) [inline src/payment/klarna-setup.md:132]:
```suggestion
8. To enable OSM in the cart page, set **Enable on Cart** to `Yes` and select a **Cart placement**.
```

File: src/payment/klarna-setup.md [hunk 5]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
 1. When complete, click <span class="btn">Save Config</span>.
 
 1. Click [**Cache Management**]({% link system/cache-management.md %}) in the message at the top of the page, and refresh all invalid caches.
+
+## Step 4: Configure On-Site Messaging (Optional)
+
+1. Activate On-Site Messaging from your Klarna Merchant Portal by following this [guide](https://developers.klarna.com/documentation/on-site-messaging/integration/platform-activation/).
+2. Retrieve your **Data Client ID**
```
Comment (ccondra) [inline src/payment/klarna-setup.md:126]:
```suggestion
2. Retrieve your **Data Client ID**.
```

File: src/payment/klarna-setup.md [hunk 6]
```diff
@@ -119,3 +119,18 @@ _Klarna Merchant Portal_
...
+3. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Klarna On-Site Messaging** section.
+4. Set **Enable** to `Yes`.
+5. Enter the **data-client-id** obtained in step 1.
+6. Choose a **Design theme** between **default** or **dark**.
+7. To enable OSM in the product page, set **Enable on Product** to **Yes** and select a **Product placement**
```
Comment (ccondra) [inline src/payment/klarna-setup.md:131]:
```suggestion
7. To enable OSM in the product page, set **Enable on Product** to `Yes` and select a **Product placement**.
```

**Pull request merged**